src/components/icons/components/content/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as BlockIcon } from './BlockIcon';
 export { default as SaveIcon } from './SaveIcon';
+export { default as AddIcon } from './AddIcon';
```

Comment:
Remember to also add export from ".src/index.ts"

Reply:
Thanks! Fixed!